Automattic/jetpack
# Boost: Show Performance History upgrade notice

Fixes Automattic/boost-cloud#362

## Proposed changes:
* Added a new Popover react component in js-packages/components
* Updated storybook config to easily develop react components in boost by allowing scss import from within boost
* Show upgrade notice for free users in Boost.

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:

<details>

![image](https://github.com/Automattic/jetpack/assets/3737780/c5788869-3038-486e-adcc-9d4d6b76472f)
<summary>Test to see if you see the upgrade prompt in performance history panel</summary>

</details>

<details>

![image](https://github.com/Automattic/jetpack/assets/3737780/29c39931-7009-470e-a85c-a0b86636234b)

<summary>Go to projects/js-packages/storybook and run `pnpm run storybook:dev` and see if Boost components are listed.</summary>

</details>
